src/probnum/filtsmooth/statespace/sde.py

```diff
@@ -82,64 +124,120 @@ class LinearSDE(SDE):
 
     def __init__(
         self,
+        dimension,
         driftmatfun: Callable[[FloatArgType], np.ndarray],
         forcevecfun: Callable[[FloatArgType], np.ndarray],
         dispmatfun: Callable[[FloatArgType], np.ndarray],
+        mde_atol=1e-6,
```

Comment:
Should we add a parameter description for `mde_atol` / `mde_rtol` to the docstring?